include/event_dbus_monitor.hpp

```diff
@@ -297,6 +297,35 @@ void registerBootProgressChangeSignal()
         BootProgressPropertyChange);
 }
 
+template <typename Callback>
+void getHiddenPropertyValue(const std::string& entryId, Callback&& callback)
+{
+    auto respHandler =
+        [callback{std::move(callback)}](const boost::system::error_code ec,
+                                        std::variant<bool>& hiddenProperty) {
+            if (ec)
+            {
+                BMCWEB_LOG_ERROR << "DBUS response error: " << ec;
+                return;
+            }
+            bool* hiddenPropValPtr = std::get_if<bool>(&hiddenProperty);
+            if (hiddenPropValPtr == nullptr)
+            {
+                BMCWEB_LOG_ERROR << "Hidden Property not found: " << ec;
+                return;
+            }
+            bool hiddenPropVal = *hiddenPropValPtr;
+            callback(hiddenPropVal);
+        };
+
+    // Get the Hidden Property
+    crow::connections::systemBus->async_method_call(
```

Comment:
I would like to avoid a call back into phosphor-logging for every new error log created by just reading the PEL interface out of the interfacesAdded signal contents, however I'm not seeing that interface in that signal so I will investigate.

Reply:
You mean we should read Hidden property directly from PEL interface - correct? @gtmills didn't like reading the property from the interface when I was providing support for CELog. He suggested to use async/callback. @gtmills any thoughts?

Reply:
> I would like to avoid a call back into phosphor-logging for every new error log created by just reading the PEL interface out of the interfacesAdded signal contents, however I'm not seeing that interface in that signal so I will investigate.

+10. InterfaceAdded looks to just have the interface name though. 
I think this is as good as we are going to get. :/ 
"Hidden" being a property instead of just a separate collection of logs has caused some really complicated code. 
I think we really need to revisit this later. 
I think either the same Redfish log service for both, and the GUI can filter, or different D-Bus collections.

Reply:
actually, give me a bit more time to debug before merging this

Reply:
OK, I got the interfacesAdded signal working:
```
signal time=1638306787.955473 sender=:1.636 -> destination=(null destination) serial=74 path=/xyz/openbmc_project/logging; interface=org.freedesktop.DBus.ObjectManager; member=InterfacesAdded                   
   object path "/xyz/openbmc_project/logging/entry/5"
   array [
      dict entry(
         string "org.open_power.Logging.PEL.Entry"
         array [                           
            dict entry(
               string "Hidden"
               variant                   boolean false
            )
            dict entry(                                                                                                                                                                                           
               string "Subsystem"           
               variant                   string "BMC Firmware"
            )    
            dict entry(
               string "ManagementSystemAck"
               variant                   boolean false
            )
         ]
      )
   ]  
```

Reply:
Then sounds like we don't need an extra call here. Thanks Matt.